kotsadm/pkg/store/ocistore/version_store.go

```diff
@@ -298,7 +298,7 @@ func (s OCIStore) CreateAppVersion(appID string, currentSequence *int64, appName
 	// NOTE that this experimental store doesn't have a tx and it's possible that this
 	// could overwrite if there are multiple updates happening concurrently
 	latestAppVersion, err := s.getLatestAppVersion(appID)
-	if err != nil && err != ErrNotFound {
+	if !s.IsNotFound(err) {
```

Comment:
still needs `err != nil` :)

Reply:
@divolgin I think this implementation of `IsNotFound` checks for `nil` in both stores, and that's a good practice.